backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Persistance/DelegationMetadataRepository.cs

```diff
@@ -57,6 +58,8 @@ public async Task<DelegationChange> InsertDelegation(DelegationChange delegation
                 pgcom.Parameters.AddWithValue("_performedByUserId", delegationChange.PerformedByUserId);
                 pgcom.Parameters.AddWithValue("_blobStoragePolicyPath", delegationChange.BlobStoragePolicyPath);
                 pgcom.Parameters.AddWithValue("_blobStorageVersionId", delegationChange.BlobStorageVersionId);
+                pgcom.Parameters.AddWithValue("_resourceid", delegationChange.ResourceId);
```

Comment:
ResourceId and ResourceType columns are now not created as nullable in 01-setup-tables.sql script.

Won't this cause the previous version of the delegation.insert_delegationchange function (which does not have these properties) to fail?

Also this function now called for an AltinnApp delegation will cause default ResourceType and an empty ResourceId to be inserted?